packages/catalog-search/src/tests/SearchPagination.test.jsx

```diff
@@ -32,13 +33,11 @@ describe('<SearchPagination />', () => {
     expect(history.location.search).toEqual('');
   });
   test('updates page query when navigating left to a previous page', () => {
-    const { history } = renderWithRouter(
-      <SearchData>
-        <SearchPaginationBase nbPages={4} currentRefinement={3} />
-      </SearchData>, {
-        route: 'search/?page=3',
-      },
-    );
+    const { history } = renderWithRouter(<SearchData>
+      <SearchPaginationBase nbPages={4} currentRefinement={3} />
+    </SearchData>, {
+      route: 'search/?page=3',
+    });
```

Comment:
This formatting looks a bit funky. Am I correct in thinking it also causes the ESLint error that you disabled on line 1 (`react/jsx-closing-tag-location`)?

Would something like the following work?

```jsx
const { history } = renderWithRouter((
  <SearchData>
    <SearchPaginationBase nbPages={4} currentRefinement={3} />
  </SearchData>,
), {
  route: 'search/?page=3',
});
```